frontend/__tests__/containers/ConfigStep/Board.test.tsx

```diff
@@ -8,36 +8,51 @@ import {
   RESET,
   VERIFIED,
   VERIFY,
-  VERIFY_ERROR_MESSAGE,
-  VERIFY_FAILED,
 } from '../../fixtures';
-import { fireEvent, render, screen, waitFor, within } from '@testing-library/react';
+import {
+  fireEvent,
+  getByLabelText,
```

Comment:
'getByLabelText' is defined but never used
BTW, 'NO_CARD_ERROR_MESSAGE' is defined but never used

Reply:
fixed